clef/clef.go

```diff
@@ -29,7 +51,7 @@ type ClefDtctr struct {
 	resultsRlfd2 chan bool
 }
 
-func NewClefDtctr(eardet *eardet.EardetDtctr, rlfd1, rlfd2 *rlfd.RlfdDtctr) *ClefDtctr {
+func NewClefDtctr(eardet *eardet.EardetDtctr, rlfd1, rlfd2 *rlfd.RlfdDtctr, t_l time.Duration, gamma, beta float64, maxWatchlistSize uint32) *ClefDtctr {
```

Comment:
According to comments above, let's double check the necessity of using `t_l` as a parameter